Source/DafnyTestGeneration/TestMethod.cs

```diff
@@ -129,10 +147,11 @@ public class TestMethod {
           return $"map[{string.Join(", ", mappingStrings)}]";
         case var arrType when new Regex("^_System.array[0-9]*\\?$").IsMatch(arrType):
           break;
+        case var _ when (!variable.Value.StartsWith("(") && variable.Value != "null"):
+          return "DATATYPES_NOT_SUPPORTED";
```

Comment:
Can you please explain me why datatypes are not supported? What was happening before?

Reply:
I have code that supports algebraic datatypes on my fork and can make another PR to add them but for know I just wanted to make it easier for a potential user to figure out why test generation might not work for them. Let me know if I should remove this case from this PR / make another PR specifically for datatypes.

Reply:
Ok thanks, It's ok to keep this case in this PR I think. I'm happy to have a look at your other PR about datatypes.